packages/FormElement/src/FormElement.js

```diff
@@ -71,85 +72,92 @@ function FormElement(props) {
     ...moreProps
   } = props;
 
-  const extratedChildren = extractChildren(children, [
+  const extractedChildren = extractChildren(children, [
     "FormElement.Description",
     "FormElement.Error",
     "FormElement.Help",
+    "FormElement.Instructions",
   ]);
   const ariaDescriptionId = React.useRef(uuidv4()).current;
-  const hasError = !!extratedChildren["FormElement.Error"] && !!extratedChildren["FormElement.Error"].props.children;
+  const ariaErrorId = React.useRef(uuidv4()).current;
+  const ariaInstructionsId = React.useRef(uuidv4()).current;
+  const hasError = !!extractedChildren["FormElement.Error"] && !!extractedChildren["FormElement.Error"].props.children;
   const uniqueInputId = React.useRef(uuidv4()).current;
   const inputId = isNil(id) || id === "" ? uniqueInputId : id;
-  let isFooterInserted = false;
 
   function renderFooter() {
-    if (isFooterInserted) return;
-
-    isFooterInserted = true;
-
     if (hasError) {
-      return extratedChildren["FormElement.Error"];
+      return React.cloneElement(extractedChildren["FormElement.Error"], {
+        ariaErrorId,
+      });
     }
 
-    if (extratedChildren["FormElement.Description"]) {
-      return React.cloneElement(extratedChildren["FormElement.Description"], {
+    if (extractedChildren["FormElement.Description"]) {
+      return React.cloneElement(extractedChildren["FormElement.Description"], {
         ariaDescriptionId,
       });
     }
 
     return null;
   }
 
+  function renderInstructions() {
+    if (extractedChildren["FormElement.Instructions"]) {
+      return React.cloneElement(extractedChildren["FormElement.Instructions"], {
+        ariaInstructionsId,
+      });
+    }
+    return null;
+  }
+
+  const nativeChildTypes = ["input", "textarea", "select"];
+
+  const renderFormElementChild = child => (
+    <div data-pka-anchor="form-element.child" key={`${child.key}`} css={formElementChildStyle}>
+      {child}
+    </div>
+  );
+
+  const ariaDescribedByIds = `${ariaErrorId} ${ariaInstructionsId} ${ariaDescriptionId}`;
```

Comment:
I'd probably prefer to put this inline – I think there's already enough abstraction/indirection with `childExtendedProps`.